lib/email/receiver.rb

```diff
@@ -1218,8 +1217,8 @@ def self.elided_html(elided)
 
     def add_other_addresses(post, sender)
       %i(to cc bcc).each do |d|
-        if @mail[d] && @mail[d].address_list && @mail[d].address_list.addresses
-          @mail[d].address_list.addresses.each do |address_field|
+        if @mail[d].present?
```

Comment:
It is nitpicking but what would you say if we introduce guard here - something like `next if @mail[d].blank?`